### PR TITLE
Hardcode the Digest Sidebar, style fixes for certain widgets therein

### PR DESCRIPTION
**Pull request by benlk**
## Changes

This pull request makes the following changes:

- Modifies ENN's `sidebar.php` to load a custom sidebar partial on the digest category page
- Adds a sidebar template to output the newly-created Digest Sidebar, or, if that is empty, nothing
- Registers a Digest Sidebar with appropriate description of where it is employed.

Screen Shot 2020-02-19 at 16 37 27
Screen Shot 2020-02-19 at 16 37 35
Screen Shot 2020-02-19 at 16 41 06

## Why

For #86

## Testing/Questions

Features that this PR affects:

- /category/digest/

Questions that need to be answered before merging:

- [x] Is this PR targeting the correct branch in this repository?
- [ ] do we want to do any specific styling for textwidget, with padding or anything like that? See https://github.com/INN/umbrella-energynewsnetwork/issues/86#issuecomment-588485946

Steps to test this PR:

1. Check out this branch
2. In Appearance > Widgets, add some widgets to the Digest Sidebar widget area. I added:
    - a text widget identifying the sidebar
    - a ENN Mailchimp signup widget
    - a textwidget that contains the following:
        ```html
        <p>
        	Are you subscribed to one or more daily digests? Add or remove subscriptions by following this link:
        </p>
        
        <a class="btn btn-primary" href="https://example.org">Manage Subscription</a>
        ```
3. Visit the following locations:
    - /category/digest/ : the Digest sidebar should be displayed
    - /category/news/ : the Digest sidebar should not be displayed
    - /region/west/ : the Digest sidebar should not be displayed
    - /category/digest/ with a search query: the Digest sidebar should be displayed

### CODE REVIEW

**Comment by joshdarby** [comment]
@benlk Should we switch the target branch to `staging` instead of `master`?

File: wp-content/themes/midwestenergynews/partials/sidebar-digest.php [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+ *
+ * Copied from Largo's partials/sidebar-archive.php at 0.6.4
+ * @package Largo
+ * @since 0.6.4
+ * @link https://github.com/INN/umbrella-energynewsnetwork/issues/86
+ */
+$custom_sidebar = largo_get_custom_sidebar();
+if ( !dynamic_sidebar( 'digest-sidebar' ) ) { // try custom sidebar registered in functions.php
+	// do nothing.
+	echo 'aaa';
```
Comment (joshdarby) [inline wp-content/themes/midwestenergynews/partials/sidebar-digest.php]: Probably want to remove this 😂

Comment (benlk) [reply wp-content/themes/midwestenergynews/partials/sidebar-digest.php]: Fixed in https://github.com/INN/umbrella-energynewsnetwork/pull/89/commits/57ec81573ab19db78bfa3b15d716018191ecfbe1 🤦‍♂

**Comment by benlk** [comment]
See updated widget screenshots in #86.

**Comment by benlk** [comment]
This is now deployed, but not yet configured with widgets.

**Comment by joshdarby** [comment]
I don't actually see this deployed on staging 😕 

And I don't have access to their WPE account to be able to deploy it.

**Comment by benlk** [comment]
Deployed for real now. Proceeding to config.

**Comment by benlk** [comment]
Configured: https://usenergystage.wpengine.com/category/digest/

**Pull request merged**